batchflow/models/torch/losses/lovasz.py

```diff
@@ -0,0 +1,147 @@
+""" Implementation of the Lovasz Softmax loss.
+Maxim Berman, et al "`The Lovász-Softmax loss: A tractable surrogate for the optimization
+of the intersection-over-union measure in neural networks <https://arxiv.org/abs/1705.08790>`_"
+
+Heavily based on author's implementation: https://github.com/bermanmaxim/LovaszSoftmax
+"""
+import torch
+from torch import nn
+import torch.nn.functional as F
+from torch.autograd import Variable
+
+
+class BinaryLovaszLoss(nn.Module):
```

Comment:
Why you didn't sort out new losses across the `binary.py` and `multiclass.py` files?

Reply:
Those loss functions are quite long 
Also, they deserve some private space